src/fmt.rs

```diff
@@ -7,6 +7,17 @@ const NEWLINE_PATTERN: &str = "\r\n";
 #[cfg(not(target_os = "windows"))]
 const NEWLINE_PATTERN: &str = "\n";
 
+pub(crate) const DEF_TABLE_ORDER: &[&str] = &[
+    "package",
+    "workspace",
+    "lib",
+    "bin",
+    "features",
+    "dependencies",
+    "build-dependencies",
+    "dev-dependencies",
+];
```

Comment:
@jplatte We'll need to mention in the changelog that `lib` and `bin` are now also sorted to be at the top.